src/Repository/PostRepository.php

```diff
@@ -12,37 +12,48 @@ class PostRepository
     public function getPost(string $id): Post
     {
         $statement = $this->connection->getConnection()->prepare(
-            "SELECT * FROM posts WHERE id = ?"
+            "SELECT title, chapo, author, content, DATE_FORMAT(lastUpdateDate, '%d/%m/%Y à %Hh%imin%ss') AS lastUpdateDate FROM posts WHERE id = ?"
         );
         $statement->execute([$id]);
 
         $row = $statement->fetch();
         $post = new Post();
         $post->title = $row['title'];
-        $post->author = $row['author'];
-        $post->chapo = $row['chapo'];
+        $post->author = $row['chapo'];
+        $post->chapo = $row['author'];
         $post->content = $row['content'];
-        // $post->lastUpdateDate = $row['date'];
+        $post->lastUpdateDate = $row['lastUpdateDate'];
 
         return $post;
     }
 
     public function getPosts(): array
     {
         $statement = $this->connection->getConnection()->query(
-            "SELECT * FROM posts"
+            "SELECT title, chapo, author, content, DATE_FORMAT(lastUpdateDate, '%d/%m/%Y à %Hh%imin%ss') AS lastUpdateDate FROM posts"
         );
         $posts = [];
         while(($row = $statement->fetch())) {
             $post = new Post();
             $post->title = $row['title'];
-            $post->chapo = $row['chapo'];
+            $post->author = $row['chapo'];
```

Comment:
private method for hydration to not repeat with the getPost and getPosts methods

private function hydratePost(array $post): Post